dev/benchmarks/complex_layout/lib/main.dart

```diff
@@ -412,7 +412,9 @@ class ItemImageBox extends StatelessWidget {
                   child: new Container(
                     decoration: new BoxDecoration(
                       backgroundColor: Colors.black54,
-                      borderRadius: 2.0
+                      borderRadius: const BorderRadius.all(
```

Comment:
since having a circular radius is the common case I think it would make sense to have BorderRadius have that as a shorthand